Build/Build.cs

```diff
@@ -246,5 +259,25 @@ class Build : NukeBuild
                     (v, path) => v.SetTargetPath(path)));
         });
 
+    Target SpellCheck => _ => _
+        .Executes(() =>
+        {
+            static void LogErrorAsWarning(OutputType outputType, string message)
+            {
+                if (outputType == OutputType.Err)
+                {
+                    Warning(message);
+                }
+                else
+                {
+                    Information(message);
+                }
```

Comment:
I have noticed that this construct prints warnings as information.. Should we change that?

Reply:
Hmm, good point. I haven't even noticed the real warnings among all the others ;-)

Reply:
Ahh.. Nevermind.. `OutputType.Err` means output from stderr.